assets/component-newsletter.css

```
@@ -19,12 +19,10 @@
   width: 100%;
 }
 
-.newsletter-form__field-wrapper .field__input {
-  padding-right: 5rem;
-}
-
-.newsletter-form__field-wrapper .field {
+.newsletter-form__field-wrapper .field-new {
```

```suggestion
.newsletter-form__field-wrapper .field{
```

Every instance of `-new` I want to ensure is removed. We can modify the classes as they are.